pkgs/development/python-modules/cherrypy-cors/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonRelaxDepsHook
+, pytestCheckHook
+, setuptools
+, httpagentparser
+, cherrypy
+}:
+buildPythonPackage rec {
+  pname = "cherrypy-cors";
+  version = "1.7.0";
+  format = "pyproject";
+
```

```suggestion
  disabled = "3.8";

```
https://github.com/cherrypy/cherrypy-cors/blob/3b56853d1b57a45de8790205ace2f0dd33029e4f/setup.cfg#L18